src/glmtools.jl

```diff
@@ -11,41 +11,122 @@ type LogLink <: Link end
 type ProbitLink <: Link end
 type SqrtLink <: Link end
 
+"""
+Link function mapping mu to eta, the linear predictor (CauchitLink response model).
+"""
 linkfun(::CauchitLink, μ) = tan(pi*(μ - oftype(μ, 0.5)))
+"""
```

Comment:
Ah, and I think it would make sense to add a line break before `"""` now that each definition + docs takes several lines.